docs/multi-organization-guide.md

```diff
@@ -0,0 +1,182 @@
+controlled process to develop, test and validate changes to wide reaching mission-critical 
+services, including but not limited to:
+- Service Control Policies changes.
+- Identity Center and IAM based Access Management Configuration changes.
+- Deployment Framework changes.
+- AWS Organization changes; including OU structure.
+- Control Tower and Account Provisioning configurations changes.
+- Centralized security service configuration changes.
+- Centralized cost management configuration changes.
+- Centralized networking changes.
+
+The following sections are written in the context of this "".
+
+## Propagating Code Changes Between ADF Installations 
+With multiple ADF configurations across multiple AWS organization there comes a new challenge to maintain repositories and its configurations across multiple environments. 
+This applies to the following repositories: 
+- aws-deployment-framework-bootstrap
+- aws-deployment-framework-pipelines
+- any other landing zone repository
+
+As the requirements of the bootstrapped resources and adf configuration evolve they need to be updated over time, and these changes ideally propagated from one Installation to the next in a coordinated, controlled fashion.
+
+With an `Environment Branching` approach it's possible to build a 'hands-off' 
+automated mechanism to Promote from a "dev" AWS Organization installation to a "prod" AWS Organization installation.
+This means that for example merging code from the "dev" branch of a repository to the "int" branch of a repository, will trigger the deployment process in the "int" AWS Organization.
+
+Implementing such an approach is out of scope of this guide and it heavily depends on the specific source code & CICD tool in use. 
+
+
+## Best Practices for Multi-Organization ADF setups
+If you want to run ADF in a multi-organization setup, there are various best practices that should be followed.
+When following these recommendations, the content of the repository `aws-deployment-framework-bootstrap` and `aws-deployment-framework-pipelines` should be stage agnostic. 
+This means that you can copy and paste the content of any of those two repositories into any AWS Organization stage ("dev", "int", "prod") and ADF will behave exactly 
+
+### 1. Create a dedicated adfconfig.yml Per AWS Organization
+One challenge with synchronising the aws-deployment-framework-bootstrap repository
+across AWS Organizations is that the contents of the `adfconfig.yml` configuration
+file is typically tailored to the ADF installation. The can be solved by adding a 
+custom adfconfig file for the given organization.
+
+Adding a configuration file with the name pattern `adfconfig.{organization id}.yml`
+in the root of the `aws-deployment-framework-bootstrap` repository will take
+precedence over the default `adfconfig.yml` settings file for that organization.
+
+For each AWS organization used with the ADF Framework setup an additional adfconfig
+file can be defined.
```

Comment:
Can you show an example, with a test organization id here?
So readers would know if they got the right type of identifier that looks the same?

Maybe you could use the same `tree` like structure of demonstrating where the files are stored like what is used in the admin guide: https://github.com/awslabs/aws-deployment-framework/blob/68fad6de334e4d9467e77e63246290f9f9032abf/docs/admin-guide.md#bootstrapping-inheritance